Documentation/adding-new-repository.md

```diff
@@ -41,6 +41,30 @@ When applicable, prepare Docker image repositories for the new component.
   * robots: Read & Write
 * For closed source repository, create a new private repository in registry.mender.io
 
+## Makefile
+
+Consider adding a Makefile to the repository. This will serve both developers on their
+local environment and automated templated jobs on CI.
+
+A Makefile in a Mender repository is expected to contain at least the following targets:
+
+* `fmt`: Format the code. Typically using `gofmt` for golang repositories
+* `lint`: Lint the code. On new golang repositories, we are trying to use `golangci-lint` tool.
+* `build`: Build repository binary in `bin/` directory.
+* `test`: Run unit tests and generate code coverage metrics in file `coverage.txt`.
```

Comment:
I think we should document `check` as well, because this is the test target for autotools. In mender-binary-delta, this is the only one that works.

Reply:
Okey. Should we document it here like `check` being an alias to `test`?

Reply:
Yes, I think that's the best compromise. `check` is historically more correct, but `test` is more modern and intuitive, so I think they stand about equally.